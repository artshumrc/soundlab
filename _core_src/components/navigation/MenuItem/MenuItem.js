import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router';


import './MenuItem.css';

const MenuItem = ({ to, onClick, children }) => (
	<Link
		className="menuItem"
		to={to}
		onClick={onClick}
	>
		{children}
	</Link>
);

MenuItem.propTypes = {
	to: PropTypes.string,
	onClick: PropTypes.func,
};

export default MenuItem;
