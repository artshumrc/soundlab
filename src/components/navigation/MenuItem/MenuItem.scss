@import '../../../variables.scss';

.menuItem {
	@include orpheus_font_body1;
	padding: 15px 20px;
	display: block;
	text-align: left;
	font-size: 12px;
	text-transform: uppercase;
	cursor: pointer;
}
