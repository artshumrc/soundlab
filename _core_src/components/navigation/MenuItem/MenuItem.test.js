import React from 'react';
import renderer from 'react-test-renderer';

// component:
import MenuItem from './MenuItem';

describe('MenuItem', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(<MenuItem />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
