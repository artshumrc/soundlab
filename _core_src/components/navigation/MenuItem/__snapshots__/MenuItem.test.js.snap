// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem renders correctly 1`] = `
<a
  className="menuItem"
  onClick={[Function]}
  style={Object {}}
/>
`;
